src/components/About/about.css

```diff
@@ -0,0 +1,106 @@
+.about-wrapper {
+  width: 340px;
+  margin: 0 auto;
+  border-radius: 20px;
+  background-color: var(--primary-element-color);
+}
+
+.about-title {
+  width: 300px;
+  display: block;
+  margin: 0 auto;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  color: var(--primary-text-color);
+}
+
+.about-text {
+  width: 300px;
+  height: auto;
+  margin: 0 auto;
+  color: var(--secondary-text-color);
+}
+
+.about-text a {
+  word-wrap: break-word;
+  transition: none;
+  color: var(--accent-text-color);
+}
+
+[data-theme='light'] {
+  .about-wrapper {
+    background-color: var(--primary-element-color);
+  }
+}
+
+[data-theme='light'] {
+  .about-title {
+    color: var(--primary-text-color);
+  }
+}
+
+[data-theme='light'] {
+  .about-text {
+    color: var(--secondary-text-color);
+  }
+}
+
+[data-theme='light'] {
+  .about-text a {
+    color: var(--accent-text-color);
+  }
+}
```

Comment:
You should use [data-theme] atributes only in main (or what-it-name) file. Where you just ajust variables. So this selectors and rules are redundant, if you did everithing ok ;)

Reply:
done